airflow/operators/sensors.py

```diff
@@ -52,10 +52,10 @@ def poke(self, context):
         raise AirflowException('Override me.')
 
     def execute(self, context):
-        started_at = datetime.now()
+        self.started_at = datetime.now()
```

Comment:
My vote goes against the use of `self` here. Object attributes created outside the constructor can be confusing.